### PR TITLE
Project Whirlwind Story - Meesue

### PR DESCRIPTION
**Pull request by meesuekim**

### CODE REVIEW

**Comment by mscuthbert** [comment]
Hi Meesue -- great work.  Some comments:

You have two photos for the story -- are they copyright cleared?  We can't add images without proper copyright permission.  Where did they come from, who was the photographer, etc.

Can you references items in our archive relating to Whirlwind?  We want to show how our archive makes a deeper connection with the Whirlwind project, so please find at least two or three documents in our archive that enhance our knowledge of the Whirlwind.

Can you rename the file to whirlwind.jinja2 or something like that?  "meesue_story" will appear in the browser address bar, so it doesn't fit the standards of professionalism we want to achieve.

THANKS!

File: templates/archives/stories/meesue_story.jinja2 [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+The Rise and Fall of Project Whirlwind
+{% endblock %}
+
```
Comment (ltagliaferri) [inline templates/archives/stories/meesue_story.jinja2]: You should add your name here :)

File: templates/archives/stories/meesue_story.jinja2 [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <a href="https://infinitehistory.mit.edu/video/edward-murrows-see-it-now%E2%80%94jay-forrester-and-whirlwind-computer-1951">
+        magnetic core memory,
+    </a>
+    which allowed for high speeds and reliability, just not enough for the computer to function properly as a
+    universal flight simulator. Because of the use of the magnetic core memory, Whirlwind became a fully functioning
+    computer that was used for scientific and military research.
+</p>
+
+<figure class="figure float-right">
+    <img src="{{ static('img/stories/meesue_story.jpg') }}"
```
Comment (ltagliaferri) [inline templates/archives/stories/meesue_story.jinja2]: This image filename should change per Myke's review.

File: templates/archives/stories/meesue_story.jinja2 [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+{% block story_credit %}
+
+{% endblock %}
+
+
+{% block story_body %}
+<p>
+    Arguably the first large-scale and high-speed computer, the development of the Whirlwind Project started in 1944 by
+    the MIT Servomechanisms Laboratory in a project directed by Jay Forrester for the United States Navy. This project
```
Comment (ltagliaferri) [inline templates/archives/stories/meesue_story.jinja2]: The grammar on this is a little off as the computer is not referred to by name but by project. I would re-phrase:

```
Arguably the first large-scale and high-speed computer, the Whirlwind was developed by the MIT Servomechanisms Laboratory in a project directed by Jay Forrester for the United States Navy.
```

You can add the fact about the date into another sentence.

File: templates/archives/stories/meesue_story.jinja2 [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    The development of Whirlwind served as a milestone in the history of computation at MIT, but it all came at a steep
+    cost. The expected cost of the project was initially $200,000, and it was projected to take approximately two years
+    to finish. Although with any project there is always the inevitability of going over budget and taking more time
+    than expected, the Whirlwind Project went over the projections far more than other computer projects of that time.
+</p>
+
+<p>
+    Starting in 1947, the Whirlwind Project received funding from the Office of Naval Research (ONR). A huge problem
+    with this project was the sheer amount of resources going into it. The research team consisted of more than one
+    hundred people, and the budget was close to $3 million. This alone took up 20% of the ONR’s total research budget.
```
Comment (ltagliaferri) [inline templates/archives/stories/meesue_story.jinja2]: Some of your colleagues have mentioned what the money at the time is now worth in today's money, which would be worth pointing out so people could better understand the magnitude of investment.

**Pull request closed without merging**